routes/post-routes.js

```diff
@@ -8,8 +8,8 @@ app.get('/', postsController.search)
 // app.get('/feed', postsController.getFollowingPosts)
 // app.get('/public', postsController.getPublicPosts)
 app.get('/id/:id', postsController.getPostByID)
-
-app.delete('/', auth.authenticate, auth.authorizedToDelete,
-  postsController.delete)
+app.post('/like/:id', auth.authenticate, postsController.likePost)
+app.delete('/unlike/:id', auth.authenticate, postsController.unlikePost)
```

Comment:
IF we stay with two different endpoints to like/unlike, then I think it should be POST `/like/:id` and DELETE `/like/:id`